### PR TITLE
Upgraded guice version and moved to jakarta annotations

### PR DESCRIPTION
**Pull request by Lucky392**

### CODE REVIEW

**Review by dsyer — CHANGES_REQUESTED** [review]
LGTM. I'll find some time to kick off a build and release if the tests are OK

File: pom.xml [hunk 1]
```diff
@@ -81,7 +86,7 @@
 		<java.version>1.8</java.version>
```
[the remark this answers is not in the document]
Comment (Lucky392) [reply pom.xml]: Updated to 17, thanks!

**Comment by dsyer** [comment]
I think you need to update the CI config in .github/maven.yml as well.

**Comment by dsyer** [comment]
Thanks. I need to fix the snapshot deployments as well. Did it manually for now.

**Comment by Lucky392** [comment]
Thank you!

**Pull request merged**